src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -265,13 +385,22 @@ class TWFFastDerivWrapper
    */
   ValueType trAB(const std::vector<ValueMatrix>& A, const std::vector<ValueMatrix>& B);
 
+  /**
+   * @brief transform X = A[o,v] - A[o,o].B[o,v]
+   * 
+   * @param[in] A matrix with dims [o,o+v]
+   * @param[in] B matrix with dims [o,v]
+   * @param[out] X matrix with dims [o,v]
+   */
+  void transform_Av_AoBv(const ValueMatrix& A, const ValueMatrix& B, ValueMatrix& X) const;
 
 private:
   std::vector<SPOSet*> spos_;
   std::vector<IndexType> groups_;
   std::vector<ValueMatrix> psi_M_;
   std::vector<ValueMatrix> psi_M_inv_;
   std::vector<WaveFunctionComponent*> jastrow_list_;
+  std::vector<const WaveFunctionComponent*> slaterdets_;
```

Comment:
Please document what this is.

Reply:
changed to `const WaveFunctionComponent*` and added a description

Reply:
If I understand correctly, thesingle det code path doesn't use this array and only the multidet uses it. For this reason, `multislaterdets` can be a better name.

Reply:
that's a good point; I've changed the name to reflect this